Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+/**
+ * Factored out common process for running test suite in python container
+ *
+ * @param pyimage Custom python image
+ */
+def test_with_container(String pyimage) {
+    withDockerServer([uri: PCIC_DOCKER]) {
+        def pytainer = docker.image(pyimage)
+
+        pytainer.inside {
+            // Installs
+            sh 'pip install -r requirements.txt -r test_requirements.txt -i http://tools.pacificclimate.org/pypiserver/ --trusted-host tools.pacificclimate.org'
+            sh 'pip install .'
+
+            // Tests
+            sh 'py.test -v --tb=short --cov --flake8'
+        }
+    }
+}
+
+
+/**
+ * Build and release package to pypi
+ */
+def push_to_pypi() {
+    withDockerServer([uri: PCIC_DOCKER]) {
+        def pytainer = docker.image('crmprtd-python35')
+
+        pytainer.inside {
+            // get twine
+            sh 'pip install twine'
+
+            // Build
+            sh 'python setup.py sdist'
```

Comment:
We should build a wheel too. So:

```
sh 'pip install twine wheel'

sh 'python setup.py sdist bdist_wheel'
```

You're `twine upload`ing everything in the dist/ directory so it should do both of them.

Reply:
Sounds good, didn't notice I missed it!